src/Stripe.net/Services/Events/EventUtility.cs

```diff
@@ -166,10 +166,22 @@ public static void ValidateSignature(string json, string stripeSignatureHeader,
 
         private static ILookup<string, string> ParseStripeSignature(string stripeSignatureHeader)
         {
+            (string Key, string Value) ParseItem(string item)
+            {
+                string[] parts = item.Trim().Split(new[] { '=' }, 2);
```

Comment:
Should that '2' be removed? How does it work with the first test case? Wouldn't that case return two parts?

Reply:
I see. Thanks!

Reply:
The first test case gets parsed into 2 elements with the second one being empty. ` ParseStripeSignature` succeeds but validation fails further down the line.